### ISSUE TITLE
Seems that BatchNorm is not converted properly

### ISSUE THREAD

**Issue by MaratZakirov** [NVIDIA-AI-IOT/torch2trt#232]
```
from torch2trt import torch2trt
import torch

bn = torch.nn.BatchNorm2d(2).cuda()
bn.eval()

x = torch.ones(1, 2, 3, 3).cuda()

# convert to TensorRT feeding sample data as input
bn.eval() # super important for bn
model_trt = torch2trt(bn, [x], max_batch_size=1)
print('Phase 2: torch2trt model generated')

y_trt = model_trt(x)
print('Phase 3: y_trt output evaluated')
y = bn(x)
print(y.shape, y_trt.shape)
y_diff = torch.abs(y - y_trt)
y_abs = (torch.abs(y) + torch.abs(y_trt)) / 2
y_diff = y_diff / (y_abs + 0.00000000001)
M_idx = (y_diff == torch.max(y_diff)).nonzero()
print('Maximum is in the:', M_idx, 'shape:', y_diff.shape, 'max:', torch.max(y_diff))
```

Creates significant difference which in my case (huge ResNet like detector) accumulated after 137 layers. If I keep only conv2d it generated ZERO difference between pytorch and tensorrt tensors.

**Comment by MaratZakirov** [comment]
It is seems to be related with this pytorch bug https://github.com/pytorch/pytorch/issues/20967

**Comment by igolan89** [comment]
@MaratZakirov Did you manage to fix it?
